include/chemfiles/Topology.hpp

```diff
@@ -129,7 +129,7 @@ class CHFL_EXPORT Topology final {
 
     /// Get the bond order for the given bond
     ///
-    /// If the bond does not exist, this will return a out_of_bonds error
+    /// If the bond does not exist, this will thrown an Error.
```

Comment:
Should this be `throw`?

Reply:
I capitalized it because it throws the `Error` class

Reply:
No, I mean throw vs thrown =)